Extensions/Leaderboards/JsExtension.js

```diff
@@ -92,6 +94,28 @@ module.exports = {
         'gdjs.evtTools.leaderboards.saveConnectedPlayerScore'
       );
 
+    extension
+      .addAction(
+        'SetPreferSendConnectedPlayerScore',
+        _('Enable the score to the connected player'),
```

Comment:
this doesn't make sense. `Enable attaching the score to the connected player`?

Reply:
Fixed